src/VisualStudio/Core/Def/Implementation/Remote/ServiceHubRemoteHostClient.cs

```diff
@@ -93,9 +94,12 @@ private enum GlobalNotificationState
                     workspace, primary.Logger, await RequestServiceAsync(primary, WellKnownServiceHubServices.SnapshotService, hostGroup, timeout, cancellationToken).ConfigureAwait(false));
                 client = new ServiceHubRemoteHostClient(workspace, primary, hostGroup, new ReferenceCountedDisposable<RemotableDataJsonRpc>(remotableDataRpc), remoteHostStream);
 
+                var uiCultureLCID = CultureInfo.CurrentUICulture.LCID;
+                var cultureLCID = CultureInfo.CurrentCulture.LCID;
+
                 // make sure connection is done right
                 var host = await client._rpc.InvokeWithCancellationAsync<string>(
-                    nameof(IRemoteHostService.Connect), new object[] { current, TelemetryService.DefaultSession.SerializeSettings() }, cancellationToken).ConfigureAwait(false);
+                    nameof(IRemoteHostService.Connect), new object[] { current, uiCultureLCID, cultureLCID, TelemetryService.DefaultSession.SerializeSettings() }, cancellationToken).ConfigureAwait(false);
```

Comment:
this is real one. and RemoteHostService set culture of OOP same as VS one which we got here.

Reply:
well. I could change CultureInfo.CurrentUICuture to CultureInfo.DefaultCurrentUICuture.

Reply:
How would that help?  If the VS culture changes, that's not going to update the `CultureInfo.DefaultCurrentUICuture`.

Reply:
I was just saying changing  CultureInfo.CurrentUICuture to CultureInfo.DefaultCurrentUICuture since someone could have changed current threads culture. 

not sure what you mean by
> How would that help? 

about 
> If the VS culture changes, that's not going to update the CultureInfo.DefaultCurrentUICuture.

you can't change VS culture without close and re-open VS, so I don't need to monitor VS culture setting change.

Reply:
Got it.  For some reason I didn't realize it required a restart.